src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int countOfFigure = 5;
+    private final int sideSize = 10;
+
+    public Figure getRandomFigure() {
```

Comment:
implement also method `Figure getDefaultFifure()` and call it from the main class according the task requirements